### PR TITLE
Uplifted the springboot version to 2.7.5

### LINKED ISSUE #30: Fix security vulnerabilities in remrem component
### Description

Given the security compliance , REM component should adhere to it.

Applicable dependencies:
Uplift of gson to 2.8.9 or later
Uplift of Jackson-databind to 2.13.3
Or more If required

Acceptance Criteria:
All security related components uplifted to the latest versions.
Test cases verified for the changes done
Function test cases verified for the changes done.
### Motivation

To overcome security issues
### Benefits

Vulnerability free code

### PR DESCRIPTION
**Pull request by vishnu-alapati**
### Description of the Change

Uplifted spring-boot version to 2.7.5 to resolve security conflicts
Uplifted spring-security-ldap version to 5.7.5
Uplifted mockito-core version to 4.5.1

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Vishnu Alapati vishnu.alapati@tcs.com

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 2.1.0
+- Uplifted spring-boot version to 2.7.5 from 2.0.1.RELEASE
```
Comment (thomas-wikstrom) [inline CHANGELOG.md]: I recommend keeping the "... from <version>  to <version> .." format as in previous comments.
This just gets harder to read when mixing formats.

**Review by thomas-wikstrom — CHANGES_REQUESTED** [review]
Pls update to continue using "from .. to .." format in CHANGELOG.md

File: CHANGELOG.md (hunk 1 above)
Comment (vishnu-alapati) [reply CHANGELOG.md]: Updated the change-log file

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 2.1.0
+- Uplifted spring-boot version from 2.0.1.RELEASE to 2.7.5.
+- Uplifted spring-security-ldap version to 5.7.5 from 5.0.5.RELEASE to 5.7.5.
```
Comment (thomas-wikstrom) [inline CHANGELOG.md]: First occurence of "to 5.7.5 " should be removed, I suppose.

**Review by thomas-wikstrom — CHANGES_REQUESTED** [review]
Pls cghg according to comment.

File: CHANGELOG.md (hunk 2 above)
Comment (vishnu-alapati) [reply CHANGELOG.md]: Thanks, updated the file

**Review by Thandava-Yelugu — APPROVED** [review]

**Review by dhruvin-kalavadia — APPROVED** [review]

**Review by thomas-wikstrom — APPROVED** [review]

**Review by jainadc9 — APPROVED** [review]

**Pull request merged**